management_api_app/tests/test_api/test_routes/test_workspaces.py

```diff
@@ -80,20 +92,26 @@ async def test_workspaces_id_get_returns_422_if_workspace_id_is_not_a_uuid(get_w
 @patch("api.dependencies.workspaces.WorkspaceRepository.get_workspace_by_workspace_id")
 async def test_workspaces_id_get_returns_workspace_if_found(get_workspace_mock, app: FastAPI, client: AsyncClient):
     sample_workspace = {
-        "id": "afa000d3-82da-4bfc-b6e9-9a7853ef753e",
+        "id": "933ad738-7265-4b5f-9eae-a1a62928772e",
         "resourceSpec": {
-            "name": "tre-workspace-vanilla",
+            "name": "My workspace",
```

Comment:
Just to make sure I'm not getting confused - this is the bundle name? And friendly name is the workspace name?

Maybe we should have them as `bundleName` and `workspaceName` to be explicit.

Reply:
Good catch - this was a copy paste from the example schema - will change both here and in the schema to tre-workspace-vanilla

Reply:
you should read it as ``resourceSpec_name``